vividus-docker-bundler/build.gradle

```diff
@@ -1,6 +1,12 @@
-plugins {
-    id 'com.bmuschko.docker-remote-api' version '9.3.4'
+buildscript {
+    repositories {
+        gradlePluginPortal()
+    }
+    dependencies {
+        classpath 'com.bmuschko:gradle-docker-plugin:9.3.4'
```

Comment:
what's the reason to apply the same plugin twice?

Reply:
I do not apply it twice, otherwise it would fail with appropriate error, here I just add it to classpath

Reply:
ok, isn't it in the classpath already?

Reply:
even though the plugin is added in vividus-test-docker.gradle, its classes are not available here, so if I remove buildscript it will complain that classes from gradle-docker-plugin are not defined